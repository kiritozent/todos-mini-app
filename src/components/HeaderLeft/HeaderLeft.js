import React from 'react';
import PropTypes from 'prop-types';
import { Typography } from 'antd';

const HeaderLeft = props => {
  const { user } = props;
  return (
    <Typography.Text className="headerLeft">
      {`Todos Mini App ${user ? ` | ${user.name}` : ''}`}
    </Typography.Text>
  );
};

HeaderLeft.propTypes = {
  user: PropTypes.object,
};

HeaderLeft.defaultProps = {
  user: null,
};

export default HeaderLeft;
