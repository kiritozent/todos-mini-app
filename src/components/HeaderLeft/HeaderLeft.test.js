import React from 'react';
import { shallow } from 'enzyme';
import HeaderLeft from './HeaderLeft';

describe('HeaderLeft', () => {
  it('render without crashing passed', () => {
    const testCase = shallow(<HeaderLeft />);
    expect(testCase).toMatchSnapshot();
  });
});
