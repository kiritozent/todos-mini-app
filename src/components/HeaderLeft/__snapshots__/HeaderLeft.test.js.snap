// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderLeft render without crashing passed 1`] = `
<Text
  className="headerLeft"
>
  Todos Mini App 
</Text>
`;
